integration/autoScalingGroup.go

```diff
@@ -0,0 +1,124 @@
+package integration
```

Comment:
Go files are conventionally all lowecase, so this file should be named autoscalinggroup.go